hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/replication/health/TestDeletingContainerHandler.java

```diff
@@ -217,7 +217,7 @@ public void testResendDeleteCommand() throws NotLeaderException {
     containerInfo = ReplicationTestUtil.createContainerInfo(
         ecReplicationConfig, 1, DELETING);
     containerReplicas = ReplicationTestUtil
-        .createReplicas(containerInfo.containerID(),
+        .createEmptyReplicas(containerInfo.containerID(),
```

Comment:
We should add a negative test case that commands are not sent for non-empty replicas.

Reply:
@smengcl has already added negative test case here 
hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/TestContainerReportHandling.java . I am not if you meant the test case there or something else.

Reply:
Yeah the integration test has it but it's good to check with a simple unit test as well.